resources/dicts/patrols/general/medcat.json

```diff
@@ -212,5 +212,80 @@
+    ],
+    "fail_text":[
+        "p_l tries to simplify the mystical power of their ancestors, but it only leaves r_c more confused about StarClan and their prophecies.",
+        null,
+        null,
+        null
+    ],
+    "min_cats": 2,
+    "max_cats": 2,
+    "win_skills": null,
+    "win_trait": ["None"],
+    "antagonize_text": null,
+    "antagonize_fail_text": null
+},
+{
+    "patrol_id": "gen_med_medwar_natureofdeath",
+    "biome": "Any",
+    "season": "Any",
+    "tags": ["med_cat", "warrior", "herb", "random_herbs", "no_app"],
+    "intro_text": "To p_l's concern, their warrior escort is unusually quiet. When prompted on their troubles, r_c comments that they could not help but think of what legacy they would leave behind if StarClan calls upon them.",
+    "decline_text": "p_l isn't sure how to exactly address such a statement, and quickly attempts to distract r_c with some herb collecting.",
+    "chance_of_success": 60,
+    "exp": 10,
+    "success_text": [
+        "p_l and r_c spend their time on the patrol discussing how to StarClan, the end of their life did not mean it was the true end. Even in StarClan, cats had their role to play. r_c's legacy would shine brighter than any star. Comforted by these words, r_c thanks p_l and helps bring back the herbs."
+    ],
+    "fail_text":[
+        "p_l attempts to soothe r_c's worries, but nothing seems to stick. r_c turns back towards the camp, leaving p_l alone to fruitlessly hunt for herbs.",
+        null,
+        null,
+        null
+    ],
+    "min_cats": 2,
+    "max_cats": 2,
+    "win_skills": null,
+    "win_trait": ["None"],
+    "antagonize_text": null,
+    "antagonize_fail_text": null
+},
+{
+    "patrol_id": "gen_med_medwar_puddles",
+    "biome": "Any",
+    "season": "Any",
+    "tags": ["med_cat", "warrior", "herb", "random_herbs", "no_app"],
+    "intro_text": "p_l could not help but be in a grumpy mood after such a storm. Herbs strewn about, prey in hiding, it would take forever to find anything. r_c attempts to find a way to cheer up the healer.",
+    "decline_text": "But before r_c could come up with a plan, a cry from the camp forces p_l to return back to camp.",
+    "chance_of_success": 60,
+    "exp": 10,
+    "success_text": [
+        "r_c splashes water from a puddle onto p_l's fur. p_l fluffs up like a bottlebrush, turning around to quickly send a wave of water back towards r_c! The two engage in a small water fight, and when they resume the herb patrol, r_c is glad to see p_l looks much happier."
+    ],
+    "fail_text":[
+        "r_c splashes water from a puddle at p_l. p_l turns around and hisses at r_c that they don't have time for kit games! The two continue their patrol, but p_l's sour mood distracts them from any usable herbs they could find.",
+        null,
+        null,
+        null
+    ],
+    "min_cats": 2,
+    "max_cats": 2,
+    "win_skills": null,
+    "win_trait": ["None"],
```

Comment:
this could be a fun patrol to give some win traits to! the outgoing traits stand out to me as good contenders, or the benevolent traits like empathetic and compassionate.